package io.pillopl.testablearch.ex2.bigballofmud.exceptions;

public class EntityNotFound extends RuntimeException {

    public EntityNotFound() {
        super();
    }
}
